### ISSUE TITLE
some missing alt text

### ISSUE THREAD

**Issue by thekuiperbelt** [chaynHQ/soulmedicine#152]
Example page: https://soulmedicine.io/en

What It Means: Two images near each other have the same alternative text.

Why It Matters: When two images have the same alternative text, this often causes redundancy or indicates incorrect alternative text.

How to Fix It: Ensure that the alternative text for each image or image button is appropriate while removing unnecessary redundancy. If the content of the image is already conveyed elsewhere (through text or the alternative text of a nearby image) or if the image does not convey content, the image may generally be given empty/null alternative text (alt=""). Image buttons always convey a specific function, and thus cannot be given null alternative text.

![image](https://user-images.githubusercontent.com/25801804/70088366-b6f8c800-15db-11ea-846f-dd2458031056.png)

**Comment by sophieheb** [comment]
This has now been fixed for all pages except for the error pages.

Awaiting PR for fix for the error pages.

**Comment by sophieheb** [comment]
Merged and fixed here: https://github.com/chaynHQ/soulmedicine/pull/160

Thank you :)
